product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -112,12 +114,56 @@ The URL contains the following information:
 
 <div id="define_oracle_url" class="registered_link"></div>
 
+## Defining an EDB Cloud URL
+
+Migration Toolkit facilitates migration from the following platforms to EDB Cloud:
+
+-   Oracle
+-   MySQL
+-   SQL Server
+-   Advanced Server
+
```

Comment:
Consider changing the order here.  I think the following order represents the "priority" of migrations we are targeting:

- Oracle
- EDB Postgres Advanced Server
- PostgreSQL
- Microsoft SQL Server
- MySQL

Reply:
+1 from me here.